### DISCUSSION TITLE
TS Documentation has wrong type

### DISCUSSION THREAD

**Discussion by geoguide** [react-hook-form/react-hook-form#2491] [General]
I was looking for the type to use in rules and the documentation shows `ValidationRules` which is not correct and does not exist, while `ValidatioOptions` was correct and is not listed in the documentation.

**Comment by bluebill1049** [comment]
could you send a PR with the fix? and where is the error?

**Comment by bluebill1049** [comment]
<img width="470" alt="Screen Shot 2020-08-06 at 12 40 34 pm" src="https://user-images.githubusercontent.com/10513364/89484523-084bfd80-d7e2-11ea-8891-8106e677e35b.png">

I think you are probably on different versions? is it V6 you are using?
